interface/ceed-operator.c

```diff
@@ -321,6 +321,22 @@ int CeedOperatorGetQFunction(CeedOperator op, CeedQFunction *qf) {
   return 0;
 }
 
+/**
+  @brief Get a boolean value indicating if the CeedOperator is composite
+
+  @param op                CeedOperator
+  @param[out] isComposite  Variable to store composite status
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Backend
+**/
+
+int CeedOperatorGetCompositeStatus(CeedOperator op, bool *isComposite) {
```

Comment:
Thanks Jeremy!
Sometimes getter functions that return a boolean are called `IsProperty()`. In this case it would be  `CeedOperatorIsComposite`; a bit shorter. This could be a minor preference for someone. It doesn't really matter to me --- overall it looks good!

Reply:
I like `Is*` better than `Get*Status`, but we use `Get*Status` pretty consistently (just an unfortunate choice of mine at some point in the past that's spread a fair bit now). I'd be open to changing it in a separate PR.

Reply:
Sounds good. 
In that case, I know this comment is also late, but perhaps we could consider changing `CeedOperatorAssembleLinearDiagonal` -> `CeedOperatorLinearAssembleDiagonal` and so on for relative functions?
It could be a minor wording preference for someone. Again, it doesn't really matter to me. But it just occurred to me that the way it is now, it sounds like the diagonal is linear (which may be confusing to someone reading it for the first time) rather than the operator

Reply:
That'd be fine with me, and best to get any such breaking changes out of the way in 0.7.

Reply:
It's possible to deprecate the old names so they continue working with a warning for the next release.

Reply:
The `Get*Status` functions are really only used internally, so the changes would be pretty easy.

Reply:
For `CeedOperator*Diagonal`, I don't really care, but now is the time to decide if we want to change the name before I tie this into MFEM.

Reply:
Valeria's naming seems more consistent if cost to change is still low.

Reply:
Cool, I can whip up a name change PR.